src/test/unit/StrategyManagerUnit.t.sol

```diff
@@ -309,6 +312,9 @@ contract StrategyManagerUnitTests is Test {
     }
 
     function testDepositIntoStrategyWithSignatureSuccessfully(uint256 amount, uint256 expiry) public {
+        // min shares must be minted on strategy
+        cheats.assume(amount >= 1e9);
+
```

Comment:
I'm pretty sure the reason we had StrategyWrapper in these tests instead of StrategyBase was to avoid doing this

Reply:
fixed